new-log-viewer/public/index.html

```diff
@@ -10,6 +10,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" crossorigin href="https://fonts.gstatic.com">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
```

Comment:
I recently learned that we may want to limit the range of font weight imported to reduce the asset sizes. What do you think?

```suggestion
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300..700&display=swap">
```

Also shall we use `@import` for font imports? e.g.
```
<style>
@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300..700&display=swap');
</style>
```

Reply:
1. I can't find one. With that said, it might be better to use a CDN for font loading rather than serving the font files directly from the log-viewer, so that loading of the log viewer can be sped up if the font has been loaded locally by another site. (It can be another story when we release the Log Viewer with Desktop UI frameworks like Electron. )
2. [Online critics](https://stackoverflow.com/questions/12316501/including-google-fonts-link-or-import) have said that `@import` defer loading of other resources, so we might not want to use `@import` given we have been managing all font loading in the same HTML file.

Reply:
1. Gotcha. I guess we can specify a weight then, when we need to add fonts.
2. Gotcha.